src/texting/xlsx-builder.js

```diff
@@ -0,0 +1,116 @@
+const excel = require('excel4node');
+const Helper = require('../helper');
+
+class ExcelBuilder {
```

Comment:
I would rather create a dedicated package for that, maybe called /reporting
Package /testing can be a place where we put all tools to format messages returned to Discord.
Creating a file I think is something a little bit different